MIGRATING-0.6-0.7.md

```diff
@@ -181,25 +231,129 @@ All `draw_*` methods to draw specific primitives (`draw_circle`, `draw_triangle`
 
 - `clear`
 
-  Fill the entire display with a solid color.
+  Fills the entire display with a solid color.
 
 These methods aim to be more compatible with hardware-accelerated drawing commands. Where possible, embedded-graphics primitives will use `fill_contiguous` and `fill_solid` to improve performance, however may fall back to `draw_iter` by default.
 
-To reduce duplication, please search the `DrawTarget` documentation on <https://docs.rs/embedded-graphics> for more details on the usage and arguments of the above methods.
+To reduce duplication, please search the `DrawTarget` documentation on <https://docs.rs/embedded-graphics-core> for more details on the usage and arguments of the above methods.
 
 ## For crates that handle images
 
-Crates that handle images should now implement items exported by the [`embedded-graphics-core`](https://crates.io/crates/embedded-graphics-core) crate to integrate with embedded-graphics.
+Crates that handle images should now implement the `ImageDrawable` and `OriginDimensions` traits to integrate with embedded-graphics.
```

Comment:
Should we replace "should" with "must", because the two traits are required to be implemented to integrate with e-g?